indico/MaKaC/webinterface/tpls/PosterTemplates.tpl

```diff
@@ -14,7 +14,7 @@
     % for template in templateList:
     <li>
         <span class="name">${template['name']}</span>
-        <div class="toolbar right thin inline">
+        <div class="right thin inline">
```

Comment:
On this page? Not necessarily. But it does look better in toolbar style.

To me the original problem looks like there's some clearing missing or some margins are too low.